es/sources_index.go

```diff
@@ -360,9 +361,31 @@ func (index *SourcesIndex) indexSource(mdbSource *mdbmodels.Source, parents []st
 			}
 			authors := authorsByLanguage[i18n.Language]
 			s := append(authors, pathNames...)
-			source.Title = s[len(s)-1]
+			leaf := s[len(s)-1]
+			source.Title = leaf
+			suffixes := Suffixes(strings.Join(s, " "))
+			if len(s) > 2 {
+				suffixes = append(suffixes, ConcateFirstToLast(s))
```

Comment:
Suffixes here will also include:
<Author> <Article Name>
?

Reply:
it resolves problems with search like הרב"ש+[Article name]. 'Rabash' is the first node and article name is the last.